spid_cie_oidc/provider/settings.py

```diff
@@ -0,0 +1,47 @@
+
+
+JWK = {
```

Comment:
@francescatronconi please move/delete/rename these constants

is there for tests purpose?

the JWK must be taken from FederationEntityConfiguration
where a openid_provider MUST be configured to get this OP to work

Reply:
@francescatronconi why this comment have been resolved?

Reply:
I have removed that and pushed on my branch of provider, but I don't see my commit here, maybe because this pull request was opened by you?

Reply:
ok